packages/flutter/lib/src/material/navigation_bar_theme.dart

```diff
@@ -0,0 +1,221 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show lerpDouble;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+
+import 'material_state.dart';
+import 'navigation_bar.dart';
+import 'theme.dart';
+
+/// Defines default property values for descendant [NavigationBar]
+/// widgets.
+///
+/// Descendant widgets obtain the current [NavigationBarThemeData] object
+/// using `NavigationBarTheme.of(context)`. Instances of
+/// [NavigationBarThemeData] can be customized with
+/// [NavigationBarThemeData.copyWith].
+///
+/// Typically a [NavigationBarThemeData] is specified as part of the
+/// overall [Theme] with [ThemeData.navigationBarTheme].
```

Comment:
This would be a good place to further explain the role that NavigationBarTheme can play (so far, I don't think any links to it show up in this API doc).

Reply:
Done